docs/template/elements.html

```diff
@@ -283,7 +283,7 @@
 </div>
 
 <!-- feedback button -->
-<div class="feedback-menu feedback-menu--techne feedback-affix"  style="display: none;">
+<div class="feedback-menu feedback-menu--techne feedback-affix">
```

Comment:
I think we need to hide this by default. if removing inline style, then we need to hide it in the CSS. 

The reason is because we are checking for a 200 (success) response from the service and then showing the feature. otherwise, the feature would not show if the service is down. Part of Wookies' resilient UI philosophy